onnxruntime/contrib_ops/cuda/moe/ft_moe/moe_gemm_kernels_template.h

```diff
@@ -453,54 +473,52 @@ void MoeGemmRunner<T, WeightType>::profile_gemm<EpilogueTag>(const T* A, const W
       chosen_config_id = ii;
     }
   }
-  best_config_map.emplace(key, candidate_configs[chosen_config_id]);
+  CutlassGemmConfig config = candidate_configs[chosen_config_id];
+  g_moe_gemm_config_map.Insert(key, config);
 }
 
 template <typename T, typename WeightType>
 template <typename EpilogueTag>
 void MoeGemmRunner<T, WeightType>::run_gemm<EpilogueTag>(const T* A, const WeightType* B, const T* weight_scales,
                                                          const T* biases, T* C, int64_t* total_rows_before_expert,
                                                          int64_t total_rows, int64_t gemm_n, int64_t gemm_k,
-                                                         int num_experts, cudaStream_t stream,
-                                                         MoEGemmConfigMap::type& best_config_map) {
-  // Generate Key to the best_config_map.
+                                                         int num_experts, cudaStream_t stream) {
+  // Generate Key to the g_moe_gemm_config_map
   // First 32 bits are total_rows, next 16 bits are gemm_n, next 16 bits are gemm_k
   int64_t key = total_rows;
   key = key << 16 | gemm_n;
   key = key << 16 | gemm_k;
```

Comment:
May need add T and WeightType to key, or use different global objects for different data type. For example, one way is 

```
template <T, WeightType>class MoeGemmRunner {
...
static MoEGemmConfigMap & GetGemmConfigMap(){
    static MoEGemmConfigMap gFactory;
    return gFactory;
   }
}
```